lib/httpoison/httpoison.ex

```diff
@@ -1,9 +0,0 @@
-defmodule ElixirAuthGithub.HTTPoison.HTTPoison do
```

Comment:
Turns out we _probably_ don't _need_ this file. 🤷‍♂ 
The **`demo`** app is working without it. ✅